ci/helpers/brokerapi_helper.go

```diff
@@ -0,0 +1,308 @@
+package helpers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/pivotal-cf/brokerapi"
+)
+
+type ByServiceID []brokerapi.Service
+
+func (a ByServiceID) Len() int           { return len(a) }
+func (a ByServiceID) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a ByServiceID) Less(i, j int) bool { return a[i].ID < a[j].ID }
+
+type ProvisioningResponse struct {
+	DashboardURL string `json:"dashboard_url,omitempty"`
+	Operation    string `json:"operation,omitempty"`
+}
+
+type LastOperationResponse struct {
+	State       string `json:"state,omitempty"`
+	Description string `json:"description,omitempty"`
+}
+
+type uriParam struct {
+	key   string
+	value string
+}
+
+func BodyBytes(resp *http.Response) ([]byte, error) {
+	buf := bytes.Buffer{}
+	_, err := buf.ReadFrom(resp.Body)
+	if err != nil {
+		return []byte{}, err
+	}
+	return buf.Bytes(), nil
+}
+
+type BrokerAPIClient struct {
+	Url               string
+	Username          string
+	Password          string
+	AcceptsIncomplete bool
+}
+
+func NewBrokerAPIClient(Url string, Username string, Password string) *BrokerAPIClient {
```

Comment:
My linter reports `func parameter Url should be URL` :(

Reply:
the parameter names should start with lowercase